Classes/MutVoteSys.uc

```diff
@@ -1238,10 +1236,16 @@ function VS_MapList LoadMapListByPrefix(string Prefix) {
 	MapName = FirstMap;
 
 	do {
-		ML.AddMap(Left(MapName, Len(MapName) - 4)); // we dont care about extension
+		ML.AddMap(CleanMapName(MapName));
 		MapName = GetMapName(Prefix, MapName, 1);
 	} until(MapName == FirstMap);
 
+	if (Right(Prefix, 1) == "-") {
+		ML.RemoveMap(Prefix$"Tutorial");
+	} else {
+		ML.RemoveMap(Prefix$"-Tutorial");
+	}
```

Comment:
I'm not a fan of this. Theoretically you could have a map just called Tutorial, without it being designed as a tutorial level for a particular game mode.

Is this necessary for anything in particular?

Reply:
It seems like if you don't specify a map list, or you specify a map list with `IncludeMapsWithPrefix=AS`, it adds all maps except `AS-Tutorial`. But, if you specify it as `IncludeMapsWithPrefix=AS-`, it will include `AS-Tutorial`.

My use case was specifically CTF, where I wanted a map list with CTF but no CTFM. So, I specified `IncludeMapsWithPrefix=CTF-`.

I've reverted this for now so you can accept the pull request. Not sure if this "CTF" vs "CTF-" thing needs to be addressed, it's just a weird edge case. And for me at least, CTF-Tutorial isn't a bad map, so I manually include it anyway. But, you wouldn't want this happening with AS-Tutorial.